### PR TITLE
Add OTR list

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: brave-lists/brave-otr.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+! List used by Brave for the OTR purposes
+!
+||ceta.tech.cornell.edu/
+||ceta.tech.cornell.edu/clinic
+||ceta.tech.cornell.edu/resources
```
Comment (ShivanKaul) [inline brave-lists/brave-otr.txt:5]: I think if you just have `||ceta.tech.cornell.edu` that should cover all these!

**Comment by ShivanKaul** [comment]
cc @pilgrim-brave this would be the list to be consumed in OTR

File: brave-lists/brave-otr.txt (hunk 1 above)
Comment (claucece) [reply brave-lists/brave-otr.txt]: Should be corrected now.

**Comment by pes10k** [comment]
@ryanbr this list isn't for filtering / blocking.  I'll link you to the spec on slack

**Comment by fmarier** [comment]
nit: we could call it just `otr.txt`, there's already a lot of "brave" in the URL :)  hxxps://github.com/**brave**/adblock-lists/**brave**-lists/**brave**-otr.txt

**Review by pes10k — APPROVED** [review]

**Pull request merged**